src/components/FeatureViews/Accounts/Accounts.scss

```diff
@@ -251,6 +304,102 @@
     color: white;
 }
 
+.editIcon {
+  position: relative;
+  top: -20px;
+  left: 10px;
+}
+
+.editResetDiv {
+  border: solid 2px $primaryColor;
+  border-radius: 10px;
+  width: 160px;
+  z-index: 2px;
```

Comment:
I don't think z-index operates by px, I think it's by number

Reply:
Updated to remove the z index, which wasn't doing anything!